### PR TITLE
Expanded docs for organizations, workspaces, and service accounts

### PR DESCRIPTION
**Pull request by tpdorsey**
Expansion of documentation for Prefect Cloud features including:

- Workspaces
- Service accounts
- Improved Prefect Cloud overview
- Expanded Organizations docs
- Overall improved linking between topics
- Badges indicating Cloud- and Org-specific features

Closes [5581](https://github.com/PrefectHQ/prefect/issues/5581)

### Example

[Preview](https://deploy-preview-7253--prefect-orion.netlify.app/ui/cloud/)

### Checklist

- [ ] This pull request references any related issue by including "closes `<link to issue>`"
	- If no issue exists and your change is not a small fix, please [create an issue](https://github.com/PrefectHQ/prefect/issues/new/choose) first.
- [x] This pull request includes tests or only affects documentation.
- [x] This pull request includes a label categorizing the change e.g. `fix`, `feature`, `enhancement`

### CODE REVIEW

File: docs/ui/workspaces.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+If the user does not already have a Prefect Cloud account, they will be able to create one when accepting the workspace collaborator invitation.
+
+To delete a workspace collaborator, select **Remove** from the menu on the left side of the user's information on this page.
+
+## Workspace sharing <span class="badge orgs"></span>
+
+Within a Prefect Cloud [organization](/ui/organizations/) organization, Admins and workspace Owners may invite users and [service accounts](/ui/service-accounts/) to work in an organization workspace. In addition to giving the user access to the workspace, the Admin or Owner assigns a [workspace role](/ui/roles/) to the user. The role specifies the scope of permissions for the user within the workspace.
+
+!!! tip "Workspace access in Prefect Cloud organizations"
+    Within an organization, users who are not Admins do not have any default access to organization workspaces. An organization Admin or workspace Owner _must_ invite users to organizations and apply workspace roles to users.
```
Comment (WillRaphaelson) [inline docs/ui/workspaces.md:90]: This is actually configurable on workspace sharing, the default access for users. Its service accounts that dont get the privileges conferred by default workspace access.

File: docs/ui/workspaces.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+In an organization workspace, select **Workspace Sharing** to manage users and service accounts for the workspace. If you've previously invited users and service accounts, you'll see them listed.
+
+![Managing sharing in a workspace in the Prefect Cloud UI.](/img/ui/workspace-sharing.png)
+
+To invite a user to become a workspace collaborator, select the Members **+** icon. You can select from a list of existing organization members. You can also enter the email address of the person you'd like to invite. If the user does not already have a Prefect Cloud account, they will be able to create one when accepting the invitation.
```
Comment (WillRaphaelson) [inline docs/ui/workspaces.md:96]: In personal accounts, you can add an arbitrary email, in orgs, you cant quite do that yet, but I will file a backlog ticket to enable. You have to add users to your org, and then your workspace.

**Review by WillRaphaelson — APPROVED** [review]

File: docs/ui/cloud.md [hunk 3]
```diff
@@ -10,29 +10,30 @@ tags:
...
-
-Features only available on Prefect Cloud include:
-
-- User accounts: personal accounts for working in Prefect Cloud. 
-- Workspaces: isolated environments for your flows, deployments, and flow runs.
-- Collaborators: invite others to work in your workspace.
-- Email notifications: configure email alerts based on flow run states and tags.
+# Welcome to Prefect Cloud <span class="badge cloud"></span>
+
+Prefect Cloud is a workflow coordination-as-a-service platform. Prefect Cloud provides all the capabilities of the [Prefect Server UI](/ui/overview/) in a hosted environment, including:
```
Comment (billpalombi) [inline docs/ui/cloud.md:15]:
```suggestion
Prefect Cloud is a workflow coordination-as-a-service platform. Prefect Cloud provides all the capabilities of the [Prefect Orion Server](https://docs.prefect.io/tutorials/orion/?h=server#running-the-prefect-server) and [UI](/ui/overview/) in a hosted environment, including:
```

File: docs/ui/organizations.md [hunk 4]
```diff
@@ -21,15 +22,41 @@ For larger teams or companies with more complex needs around user and access man
...
 See the [Prefect Cloud plans](https://www.prefect.io/pricing) to learn more about options for supporting more users, service accounts, and workspaces.
 
+## Organizations overview
+
+An organization is a type of account available on Prefect Cloud that enables more extensive and granular control over inviting workspace collaboration. Organizations are only available on Prefect Cloud.
+
+Within an organization account you can:
+
+- Invite members to join the organization.
+- Give members administrator or collaborator roles within the organization.
```
Comment (billpalombi) [inline docs/ui/organizations.md:32]: Users can be collaborators in a workspace, but not within an organization.

File: docs/ui/workspaces.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+tags:
+    - UI
+    - Prefect Cloud
+    - workspaces
+    - deployments
+---
+
+# Workspaces <span class="badge cloud"></span>
+
+A workspace is an isolated environment within Prefect Cloud for your flows and deployments. Workspaces are only available to Prefect Cloud accounts.
```
Comment (billpalombi) [inline docs/ui/workspaces.md:12]:
```suggestion
A workspace is a discrete environment within Prefect Cloud for your flows and deployments. Workspaces are only available to Prefect Cloud accounts.
```

File: docs/ui/organizations.md (hunk 4 above)
Comment (nathanielaldon) [reply docs/ui/organizations.md:32]:
```suggestion
- Give members administrator or member roles within the organization.
```

File: docs/ui/organizations.md [hunk 6]
```diff
@@ -21,15 +22,41 @@ For larger teams or companies with more complex needs around user and access man
...
 
 ![Select the Organizations icon in Prefect Cloud.](/img/ui/organizations.png)
 
-When you select an organization, you'll see an overview of the organization.
+When you select an organization, the **Profile** page provides an overview of the organization.
+
+- **Workspaces** enables you to access and manage workspaces within the organization.
+- **Members** enables you to invite and manage users who are members of the organization.
+- **Service Accounts** enables you to view, create, or edit service accounts for your organization.
+- **Roles** enabls you to configure custom roles with permission scopes for your organization.
```
Comment (nathanielaldon) [inline docs/ui/organizations.md:51]:
```suggestion
- **Roles** enables you to configure custom roles with permission scopes for your organization.
```

File: docs/ui/organizations.md [hunk 7]
```diff
@@ -81,9 +115,11 @@ Members and service accounts must already be configured for the organization. An
...
 
+See the [Workspace sharing](/ui/workspaces/#workspace-sharing) documentation for more information.
+
 ## Organization and workspace roles
 
-Prefect Cloud enables you to configure both organization and workspace roles for users.
+Prefect Cloud enables you to configure both [organization and workspace roles](/ui/roles/) for users.
 
 - Organization roles apply to users across an organization.
 - Workspace roles apply to users within a specific workspace.
```
Comment (nathanielaldon) [inline docs/ui/organizations.md:125]: Especially now that we define and differentiate via these bullets, the next line, "Select Roles within an organziation to see the configured roles for your organization" would benefit from additional clarity as well, e.g.:

"Select Roles within an organization to see the configured workspace roles for your organization"

File: docs/ui/service-accounts.md [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+- Prefect Cloud [API keys](/ui/cloud-getting-started/#create-an-api-key)
+- Organization [roles](/ui/roles/) and permissions
+
+Using service account credentials, you can [configure an execution environment](/ui/cloud-getting-started/#configure-execution-environment) to interact with your Prefect Cloud organization workspaces without a user having to manually log in from that environment. Service accounts may be created, added to workspaces, have their roles changed, or deleted without affecting organization user accounts.
+
+Select **Service Accounts** to view, create, or edit service accounts for your organization.
+
+![Viewing service accounts for an organization in Prefect Cloud.](/img/ui/service-accounts.png)
+
+Service accounts are created at the organization level, but may be shared to individual workspaces within the organization. See [workspace sharing](#workspace-sharing) for more information.
```
Comment (nathanielaldon) [inline docs/ui/service-accounts.md:27]:
```suggestion
Service accounts are created at the organization level, but individual workspaces within the organization may be shared with the account. See [workspace sharing](#workspace-sharing) for more information.
```

File: docs/ui/workspaces.md [hunk 9]
```diff
@@ -0,0 +1,104 @@
...
+
+The number of available workspaces varies by [Prefect Cloud plan](https://www.prefect.io/pricing/). See [Pricing](https://www.prefect.io/pricing/) if you need additional workspaces or users.  
+
+### Workspaces overview
+
+When you first log into Prefect Cloud, you will be prompted to create your own initial workspace. After creating your workspace, you'll be able to view flow runs, flows, deployments, and other workspace-specific features in the Prefect Cloud UI.
+
+![Viewing a workspace dashboard in the Prefect Cloud UI.](/img/ui/cloud-new-workspace.png)
+
+Select the **Prefect Logo** in the left navigation area to see all of the workspaces you can access.
```
Comment (nathanielaldon) [inline docs/ui/workspaces.md:24]: We will likely change the landing page for the Prefect Logo at some point to a more intentionally-built homepage. So when we have the opportunity like this, I think we should direct users to the "4-squares" item in the left-nav area, which is dedicated to the workspaces page (and will reliably do so in the future).

File: docs/ui/workspaces.md [hunk 10]
```diff
@@ -0,0 +1,104 @@
...
+
+To invite a user to become a workspace collaborator, select the Members **+** icon. You can select from a list of existing organization members. You can also enter the email address of the person you'd like to invite. If the user does not already have a Prefect Cloud account, they will be able to create one when accepting the invitation.
+
+Select a Workspace Role for the user. This will be the initial role for the user within the workspace. A workspace Owner can change this role at any time.
+
+Select **Send** to initiate the invitation. 
+
+To add a service account to a workspace, select the Service Accounts **+** icon. You can select from a list of existing service accounts configured for the organization. Select a Workspace Role for the service account. This will be the initial role for the service account within the workspace. A workspace Owner can change this role at any time. Select **Share** to finalize adding the service account.
+
+To delete a workspace collaborator or service account, select **Remove** from the menu on the left side of the user or service account information on this page.
```
Comment (nathanielaldon) [inline docs/ui/workspaces.md:104]:
```suggestion
To delete a workspace collaborator or service account, select **Remove** from the menu on the right side of the user or service account information on this page.
```

**Pull request merged**